.github/workflows/ci.yml

```diff
@@ -510,6 +528,80 @@ jobs:
               body: 'Failed to pass the component examples UI Test. If this PR is for UI change and the error is snapshot mismatch, please add "ui change" label to the PR for updating the snapshot.'
             })
 
+  build_upload_base_bundle_stats:
```

Comment:
To avoid pollutng the PR page with a lot of comments this would be a great candidate for the new github job summaries: https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/

Then only if the diff is great than x% do we actually post on the PR to say - wow you added a lot was this intended??